ui/apps/platform/src/Containers/Dashboard/PatternFly/Widgets/SeverityTile.css

```diff
@@ -4,3 +4,8 @@
   border-color: var(--pf-global--BorderColor--100);
   border-top-color: var(--pf-severity-tile-color);
 }
+
+.pf-severity-tile:hover {
+  border-color: var(--pf-global--BorderColor--200);
+  border-top-color: var(--pf-severity-tile-color);
+}
```

Comment:
Can you add a newline at the end of the file?

We can discuss at team meeting whether to turn on a lint rule to remind us.